pkgs/applications/editors/kakoune/plugins.nix

```diff
@@ -1,5 +1,42 @@
-{ parinfer-rust }:
+{ stdenv, parinfer-rust, fetchFromGitHub }:
 
 {
   inherit parinfer-rust;
+
+  kak-ansi = stdenv.mkDerivation rec {
```

Comment:
I think this should go in its own file and be called with `callPackage`. See weechatScripts for an example.